autoload-bootstrap.php

```diff
@@ -0,0 +1,16 @@
+<?php
```

Comment:
As per the comment about autoload in the ruleset + the comment about the paths being different in a test situation and needing to include the WPCS alias file directly for a test situation, this whole file isn't necessary.